packages/code/src/Code.spec.tsx

```diff
@@ -1,32 +1,74 @@
 import React from 'react';
-import { render, cleanup } from '@testing-library/react';
+import { render, cleanup, fireEvent } from '@testing-library/react';
 import { typeIs } from '@leafygreen-ui/lib';
 import Code from './Code';
 
+// I had to add this in to prevent another error being thrown on the click event (in addition to the TypeError)
+window.prompt = jest.fn();
+
 afterAll(cleanup);
 
 const codeSnippet = 'const greeting = "Hello, world!";';
 const className = 'test-class';
 
 describe('packages/Syntax', () => {
   describe('when multiline is true', () => {
-    const {
-      container: { firstChild: containerRoot },
-    } = render(<Code className={className}>{codeSnippet}</Code>);
+    const { container } = render(
+      <Code className={className}>{codeSnippet}</Code>,
+    );
 
-    const codeRoot = (containerRoot as HTMLElement).lastChild;
+    const codeContainer = (container.firstChild as HTMLElement).lastChild;
+    const codeRoot = (codeContainer as HTMLElement).firstChild;
+    const copyButton = codeRoot?.nextSibling?.firstChild as HTMLElement;
 
     if (!codeRoot || !typeIs.element(codeRoot)) {
       throw new Error('Multiline code element not found');
     }
 
+    if (!copyButton || !typeIs.element(copyButton)) {
+      throw new Error('Multiline copy button not found');
+    }
+
     test('root element renders as a <pre> tag', () => {
       expect(codeRoot.tagName).toBe('PRE');
     });
 
     test(`renders "${className}" in the root element's classList`, () => {
       expect(codeRoot.classList.contains(className)).toBe(true);
     });
+
+    test(`copy button actually copies text to clipboard`, () => {
```

Comment:
This is my test that's causing issues. The comments I left in the test explain the current state of things. Currently, the problem I am having is that I cannot access the contents of the clipboard to paste in to an arbitrary input element.

Reply:
I may need to pair on this to understand it.

To not hold things up I may say let's just .skip for now and revisit later.